client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,512 @@
+/**
+ * External dependencies
+ */
+import {
+	applyTestFiltersToPlansList,
+	findPlansKeys,
+	getPlan as getPlanFromKey,
+	getYearlyPlanByMonthly,
+	isMonthly,
+	TERM_MONTHLY,
+} from '@automattic/calypso-products';
+import { Button } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { getPlan, getPlanBySlug, getPlanRawPrice } from 'calypso/state/plans/selectors';
+
+/**
+ * Internal dependencies
+ */
+import './tabbed-plans-style.scss';
+function TabbedPlans( { currencyCode, onUpgradeClick, planProperties } ) {
+	const translate = useTranslate();
+	const tabList = [ 'Limited', 'Professional' ];
+
+	const featureComparisonData = {
```

Comment:
I'd move `tabList` and `featureComparisonData` out of this function as module-level constants to keep the component function cleaner. That's quite a personal preference thing so please feel free to keep them here :)

Also, in case you are interested, an interpreted language like JS often doesn't have a strong guarantee to cache constants inside a function. Thus, moving them out would potentially have a negligible memory footprint and some CPU cycles benefit. Yes, so negligible that I just want to share this like a fun fact. ;)